ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/maintenance.schema.yml

```diff
@@ -0,0 +1,109 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+allow_other_keys: true
+keys:
+  maintenance:
+    type: dict
+    display_name: Profiles and units
+    keys:
+      default_interface_profile:
+        type: str
+      default_bgp_profile:
+        type: str
+      default_unit_profile:
+        type: str
+      interface_profiles:
+        type: list
+        primary_key: name
+        convert_types:
+        - dict
+        display_name: Interface Profile Names
```

Comment:
This is strictly speaking not a list of names but a list of the full profiles. So we can just keep the key name alone.
```suggestion
```